frontend/common/src/helpers/formUtils.ts

```diff
@@ -26,19 +26,35 @@ export const unpackIds = (
 export const currentDate = (): string => new Date().toISOString().slice(0, 10);
 
 /**
- * Converts enum to a list of options for select input.
- * @param list
+ * Converts a string enum to a list of options for select input.
+ * @param list - Then string enum to convert to options
+ * @param sortOrder - An optional array to indicate desired sort order
  * @returns Option
  */
-export function enumToOptions<T>(
-  list: T,
-): { value: string | number; label: string }[] {
+export function enumToOptions(
```

Comment:
I couldn't figure out how to keep the generic T and have both parameters.  I feel that the generic wasn't adding much to the original function and using Record<string, string> is pretty descriptive while keeping the same call signature.